src/pysentimentanalyzer/sentiment_score_plot.py

```diff
@@ -17,7 +26,38 @@ def sentiment_score_plot(df, col):
 
     Examples
     --------
-    >>> plot = sentiment_score_plot(df, "reviews")
+    >>> plot = sentiment_score_plot(df, "column_name")
     >>> plt.show()
     """
-    pass
\ No newline at end of file
+
+    # Call the helper function to compute the sentiment and score
+    data = get_sentiment_and_score(df, col)
+
+    # Check that the column exists in dataframe
+    assert (
```

Comment:
We should probably not use an `assert` here but rather an `if` statement to check if the column name is in the DataFrame.

Reply:
Hi Eric, can you please let me know the thought process behind it to understand why we wouldn't use an assert within a function. I assume you are requesting to use if-else and raise exception error. Then the exception should be detected in the pytests?
Any drawbacks of having an assert statement instead?

Reply:
I have made this change and pushed the updated commit